packages/store-sync/src/blockLogsToStorage.ts

```diff
@@ -151,21 +151,17 @@ export function blockLogsToStorage<TConfig extends StoreConfig = StoreConfig>({
           };
         }
 
-        if (log.eventName === "StoreSetField") {
-          const fieldName = fieldNames[log.args.schemaIndex] as string & keyof Value<TConfig, keyof TConfig["tables"]>;
-          const fieldValue = decodeField(valueAbiTypes[log.args.schemaIndex], log.args.data) as Value<
-            TConfig,
-            keyof TConfig["tables"]
-          >[typeof fieldName];
+        if (log.eventName === "StoreSpliceRecord") {
           return {
             log,
             address: getAddress(log.address),
             namespace: table.namespace,
             name: table.name,
-            type: "SetField",
+            type: "SpliceRecord",
             key,
-            fieldName,
-            fieldValue,
+            data: log.args.data,
+            start: log.args.start,
+            deleteCount: log.args.deleteCount,
```

Comment:
Is there a way we could perform the steps to combine the new bytes with an existing record in here? Or would that prevent some optimizations we can do downstream (like encode+splice+decode in a single db tx)?